modules/examples/src/main/java/com/ibm/cloud/platform_services/iam_policy_management/v1/IamPolicyManagementExamples.java

```diff
@@ -0,0 +1,347 @@
+
+  static {
+    System.setProperty("IBM_CREDENTIALS_FILE", "../../iam_policy_management.env");
+  }
+
+  @SuppressWarnings("checkstyle:methodlength")
+  public static void main(String[] args) throws Exception {
+    IamPolicyManagement service = IamPolicyManagement.newInstance();
+
+    // Load up our test-specific config properties.
+    Map<String, String> config = CredentialUtils.getServiceProperties(IamPolicyManagement.DEFAULT_SERVICE_NAME);
+    testAccountId = config.get("TEST_ACCOUNT_ID");
+
+    try {
+      // begin-create_policy
+      ResourceAttribute resourceAttributeAccount = new ResourceAttribute.Builder()
+              .name("accountId")
+              .value(testAccountId)
+              .operator("stringEquals")
+              .build();
+
+      ResourceAttribute resourceAttributeService = new ResourceAttribute.Builder()
+              .name("serviceName")
+              .value(TEST_SERVICE_NAME)
+              .operator("stringEquals")
+              .build();
+
+      ResourceAttribute resourceAttributeResource = new ResourceAttribute.Builder()
+              .name("resource")
+              .value("SDK-Java-Test")
+              .operator("stringEquals")
+              .build();
+
+      ResourceTag resourceAttributeTag = new ResourceTag.Builder()
+              .name("project")
+              .value("prototype")
+              .operator("stringEquals")
+              .build();
+
+      SubjectAttribute subjectAttributeModel = new SubjectAttribute.Builder()
+              .name("iam_id")
+              .value(TEST_USER_ID)
+              .build();
+
+      PolicyResource policyResourceModel = new PolicyResource.Builder()
+              .attributes(new ArrayList<ResourceAttribute>(Arrays.asList(resourceAttributeAccount,
+                      resourceAttributeService, resourceAttributeResource)))
+              .tags(new ArrayList<ResourceTag>(Arrays.asList(resourceAttributeTag)))
+              .build();
+
+      PolicyRole policyRoleModel = new PolicyRole.Builder()
+              .roleId("crn:v1:bluemix:public:iam::::role:Viewer")
+              .build();
+
+      PolicySubject policySubjectModel = new PolicySubject.Builder()
+              .attributes(new ArrayList<SubjectAttribute>(Arrays.asList(subjectAttributeModel)))
+              .build();
+
+      CreatePolicyOptions options = new CreatePolicyOptions.Builder()
+              .type("access")
+              .subjects(Arrays.asList(policySubjectModel))
+              .roles(Arrays.asList(policyRoleModel))
+              .resources(Arrays.asList(policyResourceModel))
+              .build();
+
+      Response<Policy> response = service.createPolicy(options).execute();
+      Policy policy = response.getResult();
+
+      System.out.println(policy);
+      // end-create_policy
+
+      testPolicyId = policy.getId();
+    } catch (ServiceResponseException e) {
+      logger.error(String.format("Service returned status code %s: %s\nError details: %s",
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [FS: Format string should use %n rather than n (VA_FORMAT_STRING_USES_NEWLINE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#fs-format-string-should-use-n-rather-than-n-va-format-string-uses-newline).
This format string includes a newline character (\n). In format strings, it is generally  preferable to use %n, which will produce the platform-specific line separator.